src/service/standard-actions-impl.js

```diff
@@ -174,7 +174,9 @@ export class StandardActions {
         return this.handleScrollTo_(invocation);
 
       case 'goBack':
-        Services.historyForDoc(this.ampdoc).goBack();
+        Services.historyForDoc(this.ampdoc).goBack(
+          /*canPerforWindowBack*/ !!args && args['navigate']
```

Comment:
Nit: `Perfor`. And `!!` is not necessary.

Reply:
I believe the `!!` is necessary for typechecking reasons in case it may be undefined

Reply:
aaand I'm wrong. Removed